config/google_cloud_exporter/hadoop/config.yaml

```diff
@@ -0,0 +1,67 @@
+receivers:
+  jmx:
+    jar_path: ./opentelemetry-java-contrib-jmx-metrics.jar
+    endpoint: localhost:8004
+    target_system: hadoop,jvm
+    collection_interval: 60s
+    properties:
+      # Attribute 'endpoint' will be used for generic_node's node_id field.
+      otel.resource.attributes: endpoint=localhost:8004
+
+processors:
+  resourcedetection:
+    detectors: ["system"]
+    system:
+      hostname_sources: ["os"]
+
+  resourceattributetransposer:
+    operations:
+      - from: "host.name"
+        to: "agent"
+
+  # Used for Google generic_node mapping.
+  resource:
+    attributes:
+    - key: namespace
+      value: "hadoop"
+      action: upsert
+    - key: location
+      value: "global"
+      action: upsert
+
+  normalizesums:
+
+  batch:
```

Comment:
These (normalize sums, batch) are built into the version of `googlecloud` exporter we now use, which is a wrapped version of the google exporter. (https://github.com/observIQ/observiq-otel-collector/tree/main/exporter/googlecloudexporter)